clients/mobile/build-ios.sh

```diff
@@ -1,34 +1,35 @@
 #!/bin/bash
 
+# Build the dylib file
+cargo build
+
+# Create bindings
+cargo run --bin uniffi-bindgen generate --library ../../target/debug/libmobile.dylib --language swift --out-dir ./bindings
```

Comment:
**Suggestion:** Consider adding error handling for the `cargo build` and `cargo run` commands to catch any potential issues during the build process. [best practice]
```suggestion
#!/bin/bash

# Build the dylib file
cargo build || { echo "Cargo build failed"; exit 1; }

# Create bindings
cargo run --bin uniffi-bindgen generate --library ../../target/debug/libmobile.dylib --language swift --out-dir ./bindings || { echo "Bindings generation failed"; exit 1; }
```